Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:16.11-alpine3.14
+FROM node:16.19.0-alpine
```

Comment:
Can we specify the alpine version too? 
`FROM node:16.19.0-alpine3.16`

Reply:
Done